templates/librarylink.m.in

```diff
@@ -66,6 +66,7 @@ fsDefaultSettings = {
       higgs3loopCorrectionAtAtAt -> 1,   (* FlexibleSUSY[29] *)
       higgs4loopCorrectionAtAsAsAs -> 1, (* FlexibleSUSY[30] *)
       loopLibrary -> 0,                  (* FlexibleSUSY[31] *)
+      calculateAMM -> 1.5,               (* FlexibleSUSY[32] *)
```

Comment:
I think we should change this to default value to 2, for consistency.

Reply:
Fixed in 7f065affb8eb513113d9d901a112eb1028a27d78.

Reply:
Thanks!